app/views/taxons/show.html.erb

```diff
@@ -1,5 +1,14 @@
 <%= display_header title: taxon.title, breadcrumbs: [:taxons, taxon] %>
 
+
+<%= link_to 'Delete taxon', taxon_path(taxon.content_id),
```

Comment:
👍  done. This turns the button grey because of how we're styling button groups in the header, but I think it's probably better to make this functionality less prominent.